pkg/scheduler/core/extender.go

```diff
@@ -116,7 +116,7 @@ func (h *HTTPExtender) IsIgnorable() bool {
 // SupportsPreemption returns if a extender support preemption.
 // A extender should have preempt verb defined and enabled its own node cache.
 func (h *HTTPExtender) SupportsPreemption() bool {
-	return len(h.preemptVerb) > 0
+	return h.preemptVerb == "preempt"
```

Comment:
preemptVerb is configurable and does not need to be "preempt".

Reply:
But, what are the possible preemptVerb values?

Reply:
It could be anything that extender has been configured to. For example, in my extender configuration I can set "preempt" to be "Preempt" or what I want it to be.

Reply:
I think the configuration file should be a definite field instead of an arbitrary foo or bar.And I don't quite understand why it is an arbitrary string.

Reply:
Providing the flexibility does not cost us anything. Why should be rigid? That said, that's how our APIs are and we cannot suddenly change them. It would break existing extenders.

Reply:
OK, make sense.